setup_deps.py

```diff
@@ -553,20 +546,15 @@ def setup_render_pipeline():
     sp.check_call([sys.executable, "setup.py"], cwd=osp.join(project_dir, "thirdParty", "RenderPipeline"))
 
 
-def setup_mujoco_py():
-    # Set up mujoco-py (doing it via pip caused problems on some machines)
-    sp.check_call([sys.executable, "setup.py", "install"], cwd=osp.join(project_dir, "thirdParty", "mujoco-py"))
-
-
 def setup_pytorch_based():
     # Locally build PyTorch>=1.7.0 requires dataclasses (does not harm when using pytorch from pip)
     sp.check_call([sys.executable, "-m", "pip", "install", "-U", "--no-deps", "dataclasses"])
     # Set up GPyTorch without touching the PyTorch installation (requires scikit-learn which requires threadpoolctl)
     sp.check_call([sys.executable, "-m", "pip", "install", "-U", "--no-deps", "threadpoolctl"])
     sp.check_call([sys.executable, "-m", "pip", "install", "-U", "--no-deps", "scikit-learn"])
-    sp.check_call([sys.executable, "-m", "pip", "install", "-U", "--no-deps", "gpytorch"])
+    sp.check_call([sys.executable, "-m", "pip", "install", "-U", "gpytorch"])
```

Comment:
why did you do that? I remember the `--no-deps` flag to be crucial, since if you would run the command triggering `setup_pytorch_pased()` first, you would install `torch` with some default setting that we might not want

Reply:
All of those dependencies broke somewhere, because they required some secondary code that wasn't loaded in. None touched torch, so I assumed that this was some legacy stuff.

Reply:
OK, I'll trust you on this